frontend/src/TooljetDatabase/Forms/EditColumnForm.jsx

```diff
@@ -0,0 +1,226 @@
+
+  const darkDisabledBackground = '#1f2936';
+  const lightDisabledBackground = '#f4f6fa';
+  const lightFocussedBackground = '#fff';
+  const darkFocussedBackground = 'transparent';
+  const lightBackground = '#fff';
+  const darkBackground = 'transparent';
+
+  const darkBorderHover = '#dadcde';
+  const lightBorderHover = '#dadcde';
+
+  const darkDisabledBorder = '#3a3f42';
+  const lightDisabledBorder = '#dadcde';
+  const lightFocussedBorder = '#dadcde';
+  const darkFocussedBorder = '#3e63dd !important';
+  const lightBorder = '#dadcde';
+  const darkBorder = '#3a3f42 !important';
+  const dropdownContainerWidth = '360px';
+
+  const darkMode = localStorage.getItem('darkMode') === 'true';
+
+  const customStyles = tjdbDropdownStyles(
+    darkMode,
+    darkDisabledBackground,
+    lightDisabledBackground,
+    lightFocussedBackground,
+    darkFocussedBackground,
+    lightBackground,
+    darkBackground,
+    darkBorderHover,
+    lightBorderHover,
+    darkDisabledBorder,
+    lightDisabledBorder,
+    lightFocussedBorder,
+    darkFocussedBorder,
+    lightBorder,
+    darkBorder,
+    dropdownContainerWidth
+  );
+
+  const handleTypeChange = (value) => {
+    setDataType(value);
+  };
+
+  const handleCreate = async () => {
+    const colDetails = {
+      column: {
+        column_name: selectedColumn?.Header,
+        data_type: selectedColumn?.dataType,
+        ...(columnName !== selectedColumn?.Header ? { new_column_name: columnName } : {}),
+        ...(defaultValue?.length > 0 ? { column_default: defaultValue } : {}),
+        ...(nullValue !== isNotNull
+          ? {
+              constraints_type: {
+                is_not_null: isNotNull,
+              },
+            }
+          : {}),
+      },
+    };
+
+    if (columnName !== selectedColumn?.Header || defaultValue?.length > 0 || nullValue !== isNotNull) {
+      setFetching(true);
+      const { error } = await tooljetDatabaseService.updateColumn(organizationId, selectedTable.table_name, colDetails);
+      setFetching(false);
+      if (error) {
+        toast.error(error?.message ?? `Failed to edit a column in "${selectedTable.table_name}" table`);
+        onClose && onClose();
+        return;
+      }
+    }
+    tooljetDatabaseService.viewTable(organizationId, selectedTable.table_name).then(({ data = [], error }) => {
+      if (error) {
+        toast.error(error?.message ?? `Error fetching columns for table "${selectedTable}"`);
+        return;
+      }
+
+      if (data?.result?.length > 0) {
+        setColumns(
+          data?.result.map(({ column_name, data_type, keytype, ...rest }) => ({
+            Header: column_name,
+            accessor: column_name,
+            dataType: data_type,
+            isPrimaryKey: keytype?.toLowerCase() === 'primary key',
+            ...rest,
+          }))
+        );
+      }
+    });
+    toast.success(`Column edited successfully`);
+    onClose && onClose();
+  };
+
+  // const handleCreate = async () => {
```

Comment:
can remove the commented code

Reply:
I think abdul might have updated the code. Once nullable fields changes are updated to UX revamp branch we can look into it to remove the code